api/api.go

```diff
@@ -261,8 +266,10 @@ func New(auth Authority) RouterHandler {
 	return &caHandler{}
 }
 
+// Route defines routing for the API.
 func Route(r Router) {
 	r.MethodFunc("GET", "/version", Version)
+	r.MethodFunc("GET", "/capabilities", Capabilities)
```

Comment:
Is this the path of an endpoint commonly used on other services?

Reply:
I'm not sure this is the right word. If I search for capabilities, the results appear to be more related to policies than something like this. Linux, Vault, ...